data/migrations/20190323121447_createProjects.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable("projects", tbl => {
+    tbl.increments();
+    tbl.string("project_name").notNullable();
+    tbl.text("project_description");
+    tbl.boolean("project_completed").defaultTo(false);
```

Comment:
While using this methodology of `project_*` is a viable way of avoiding column overrides while joining tables with columns that are named the same way, it isn't super pretty while using the table alone in JavaScript. Saying `project.project_name` is redundant compared to `project.name`;  why not shoot for that and deal with the join stuff when it is required? It's not too repetitive with aliases for your tables, or if you have them in separate objects (like you do in your get by id for projects).